packages/lib-react-components/src/MovableModal/MovableModal.js

```diff
@@ -0,0 +1,111 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Rnd } from 'react-rnd'
+import ResizeIcon from './components/ResizeIcon'
+import withLayer from '../helpers/withLayer'
+import { Modal } from '../Modal'
+
+function MovableModal (props) {
+  const {
+    children,
+    closeFn,
+    headingBackground,
+    pad,
+    rndProps: {
+      minHeight,
+      minWidth,
+      position,
+      ...restRndProps
+    },
+    title,
+    ...rest
+  } = props
+
+  return (
+    <Rnd
+      cancel='.subtaskpopup-element-that-ignores-drag-actions'
+      enableResizing={{
+        bottom: true,
+        bottomLeft: false,
+        bottomRight: true,
+        left: true,
+        right: true,
+        top: true,
+        topLeft: false,
+        topRight: false
+      }}
+      default={position}
+      minHeight={minHeight}
+      minWidth={minWidth}
+      resizeHandleComponent={{ bottomRight: <ResizeIcon /> }}
+      resizeHandleStyles={{
+        bottom: {},
+        bottomLeft: {
+          bottom: 0,
+          left: 0
+        },
+        bottomRight: {
+          bottom: 0,
+          height: '20px',
+          right: 0,
+          width: '14px'
+        },
+        left: {},
```

Comment:
I believe these empty objects can be removed for the sides that are not being used in the `resizeHandleStyles`.